app/dcr_api/biz/service/job_service.go

```diff
@@ -278,3 +277,8 @@ func (g *JobService) uploadFile(reader io.Reader, remotePath string, compress bo
 	}
 	return nil
 }
+
+func getBucket() string {
+	env := os.Getenv("ENV")
```

Comment:
Please print warning/error when ENV is not set.

Reply:
only logs error when ENV is not set